File: IOT/anshulgurjar53_AnshulSinghGurjar_2024it1193_2/FunctionAndArrays/DecimalToAnyBase.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      Scanner scn = new Scanner(System.in);
      int n = scn.nextInt();
      int b = scn.nextInt();
      int dn = getValueInBase(n, b);
      System.out.println(dn);
   }
  
   public static int getValueInBase(int n, int b){
       // write code here
       
      int s=0,c=0; 
       while(n>0)
       {
           
           int a=n%b;
           s=(int)(a*Math.pow(10,c++))+s;
           n/=b;
           
       }
       return s;
   }
  }

File: Web/sneha-jais_snehaJaiswal_2024cs1084_2/FunctionAndArrays/decimal_to_any_base.cpp
#include<iostream>
using namespace std;
int DecToAny(int n, int b) {
//write your code here

int a[100],i=0;
while(n>0){
    a[i]=n%b;
    n=n/b;
    i++;
}
for(int j=i-1;j>=0;j--){    
cout<<a[j];
}
return 0;
}
int main() {
  int n;
  int b;
  cin >> n;
  cin >> b;
  DecToAny(n, b);
 
}